scripts/install/usr/libexec/cortx-motr/motr-server

```diff
@@ -58,7 +57,7 @@ be_log_resize() {
    m0_log "Given BE log size: $new_be_log_size"
 
    # Validate current BE LOG file size and New BE LOG file size
-   current_be_log_size=$(ls -l "$MD_DIR"/m0d-"$local_ios_fid"/db-log/o/0:28 | awk '{ print $5 }')
+   current_be_log_size=$(ls -l "$MD_DIR"/"$service_fid"/db-log/o/0:28 | awk '{ print $5 }')
    if [[ $current_be_log_size == "" ]]; then
       m0_log "Failed to get existing be log size"
       return 0
```

Comment:
Return error from here, as m0d too won't work if log is not there.

Reply:
Done.